apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.css

```diff
@@ -6,8 +6,22 @@
 
 .pl-rich-text-editor-container:focus-within {
   outline: 0.2rem solid rgb(0 123 255 / 25%);
+  box-shadow: unset;
 }
 
 .pl-rich-text-editor-container {
   box-shadow: 3px 3px 3px grey;
 }
+
+.pl-rich-text-editor-counter-container-disabled {
+  display: none;
+}
+
+.pl-rich-text-editor-counter-container {
+  border-style: solid;
+  border-color: #ccc;
+  border-width: 0 1px 1px;
+  color: #aaa;
```

Comment:
This is too light for accessibility standards (https://webaim.org/resources/contrastchecker/)

Reply:
Does PL have a standardized color palette? (I think bootstrap?)

Reply:
If you can use bootstrap may be better. Maybe text-info?

Reply:
text-info was blue, so I went with text-secondary